src/main/java/com/demoqa/pages/RadioButtonPage.java

```diff
@@ -0,0 +1,29 @@
+package com.demoqa.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class RadioButtonPage {
+    private static final String LABEL_FOR_YES_RADIO = "//label[@for='yesRadio']";
+    private static final String SCROLL_INTO_VIEW_SCRIPT = "arguments[0].scrollIntoView(true)";
+    private final WebDriver driver;
```

Comment:
The same, this way looks better
```suggestion
    private static final String LABEL_FOR_YES_RADIO = "//label[@for='yesRadio']";
    private static final String SCROLL_INTO_VIEW_SCRIPT = "arguments[0].scrollIntoView(true)";
    
    private final WebDriver driver;
    private final JavascriptExecutor js;
```